packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-input.vue

```diff
@@ -28,7 +28,7 @@ const props = defineProps<{
 	contextInfo: any;
 }>();
 
-const emit = defineEmits(['llm-output']);
+const emit = defineEmits(['output-code']);
```

Comment:
Any particular reason for the name change? I was thinking this may not necessarily be outputting code based on the props passed in?

Reply:
The intention of this is to have LLM output. Currently this is only registered to `code_cell` responses but thats mostly because i still need to work on this component. 
It should be llm-output IMO

Reply:
I was making the passed emit names consistent in `tera-model-edit`. I guess they can be different though